### DISCUSSION TITLE
Access headers from inbound function call?

### DISCUSSION THREAD

**Discussion by prescience-data** [supabase/supabase#3179] [Questions] [answered]
I'm not sure if I missed this somewhere in the docs, but was wondering if there is a way to access the headers sent by a client when accessing an RPC function?

Use case is authenticating Stripe webhooks, which attach a `stripe-signature` header.

**Comment by steve-chavez** [comment, marked as answer]
Yes, with:

```sql
SELECT current_setting('request.header.stripe-signature', true);
```

(From https://postgrest.org/en/v8.0/api.html#accessing-request-headers-cookies-and-jwt-claims)
